app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -53,6 +53,7 @@ def show; end
           param :bookmark_id, Integer, :required => false
           param :search_query, Integer, :required => false
           param :description_format, String, :required => false, :desc => N_('Override the description format from the template for this invocation only')
+          param :timeout_interval, Integer, :required => false, :desc => N_('Override the timeout interval from the template for this invocation only')
```

Comment:
One last thing: what about being more explicit with `execution_timeout_interval`? To make it clear it's not a connection timeout interval (and in case we wanted to expose the connection timeout later)